src/lib.rs

```diff
@@ -84,3 +84,6 @@ pub fn get_versions() -> Versions {
 pub fn start() {
     console_error_panic_hook::set_once();
 }
+
+#[cfg(test)]
+wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_node_experimental);
```

Comment:
This is to enable ES modules in node in the wasm_bindgen test environment. Else it won't work as we switched the package to be a module

Reply:
Do we still need this, now that the package is dual commonjs/esm?

Reply:
It *was* necessary because I've flipped the main module type before to ESM (`"type": "module"` in the package.json) which made it that `.js` files were considered as ESM not CJS, which broke the regular test.

Reply:
This was reverted